src/ui/SearchInterface/SearchInterface.ts

```diff
@@ -529,7 +529,7 @@ export class SearchInterface extends RootComponent implements IComponentBindings
     this.queryController = new QueryController(element, this.options, this.usageAnalytics, this);
     this.facetValueStateHandler = new FacetValueStateHandler((componentId: string) => this.getComponents(componentId));
     new SentryLogger(this.queryController);
-    new MissingTermManager(this.element, this.queryStateModel, this.queryController);
+    new MissingTermManager(this.element, this.queryStateModel, this.queryController, this.usageAnalytics);
```

Comment:
This signature is getting pretty big, might be better to pass in maybe the reference to the search interface itself ?